src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -120,8 +135,14 @@ public Cache createCache(@RequestBody Cache cache) {
      */
     @DeleteMapping("/cache/{id}/")
     public ResponseEntity deleteCache(@PathVariable("id") int id){
+        try {
+            //TODO: cacheService.delete(id);
+            return new ResponseEntity(HttpStatus.OK);
+        } catch (Exception e) {
+            logger.log(Level.WARNING, "Unable to delete cache with ID: " + id + ", message: " + e.getMessage());
+            return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
```

Comment:
Added Try catch and logger to help identify when the delete method is implemented if the correct id is being identified to be deleted.